nuxeo-data-table/iron-data-table.html

```diff
@@ -744,22 +744,40 @@ <h2>[[i18n('command.add')]]</h2>
       _validateEntry: function() {
         var dtform = this.getContentChildren('#form')[0];
         if (dtform.validateItem()) {
+          var item = this._deepCopy(dtform.item);
           if (dtform.index > -1) {
-            this.set('items.' + dtform.index, dtform.item);
-            //this.items[dtform.index] = JSON.parse(JSON.stringify(dtform.item));
+            this.set('items.' + dtform.index, item);
           } else {
-            this.push('items', dtform.item);
+            this.push('items', item);
           }
           this.notifyResize();
           this.$.dialog.close();
         }
       },
 
+      _deepCopy: function(obj) {
+        var cache = [];
+        var result = JSON.parse(JSON.stringify(obj, function(key, value) {
+            if (typeof value === 'object' && value !== null) {
```

Comment:
this is not formatted with 2 spaces, right?

Reply:
not really :)